Formula/juise.rb

```diff
@@ -24,12 +24,22 @@ class Juise < Formula
   depends_on "sqlite"
 
   def install
-    system "sh ./bin/setup.sh" if build.head?
+    system "sh", "./bin/setup.sh" if build.head?
+
+    # Prevent sandbox violation
+    inreplace "configure",
+      "SLAX_EXTDIR=\"`$SLAX_CONFIG --extdir | head -1`\"",
```

Comment:
I guess another option would be to vendor `libslax` into `juise`'s `libexec`, which would make the problem moot. The `juise` configure already has
```
  --with-libslax-prefix=PFX           Specify location of libslax config
  --with-libslax-include-prefix=PFX   Specify location of libslax headers
  --with-libslax-libs-prefix=PFX      Specify location of libslax libs
  --with-libslax-src=DIR              For libslax thats not installed yet (sets all three above)
```
So a vendored libslax should be trivial. But I guess that's overkill?